[Install] QEMU CPU model error

Please answer these questions before submitting your issue. Thanks!


### What version of Go are you using (`go version`)?


### Does this issue reproduce with the latest release?


### What operating system and processor architecture are you using (`go env`)?


### What did you do?

If possible, provide a recipe for reproducing the error.
A complete runnable program is good.
A link on play.golang.org is best.


### What did you expect to see?


### What did you see instead?
```bash
SeaBIOS (version 1.11.0-2.el7)


iPXE (http://ipxe.org) 00:03.0 C980 PCI2.10 PnP PMM+BFF94780+BFED4780 C980
                                                                               


Booting from Hard Disk...
 LAPIC Mhz: 00000000000003E7 
 CPU Mhz: 000000000000095A 
 no random!
Reserved 65536 pages (256MB)
              BiscuitOS
          go version: go1.10.1
  4763 MB of physical memory
CPUID: family: 6, model: 1a
sse3 true, ssse3 true, sse41 true, sse42 true, avx false
invariant TSC not supported
panic: CPU leaf 0x7 unsupported

goroutine 1 [running]:
main.cpuid(0x7, 0x1c, 0x0)
        /home/zhangyao/biscuit/biscuit/src/kernel/main.go:1029 +0x148
main.cpuchk()
        /home/zhangyao/biscuit/biscuit/src/kernel/main.go:1079 +0x2e1
main.main()
        /home/zhangyao/biscuit/biscuit/src/kernel/main.go:1644 +0x1b1
 exit with code 0000000000000002 .
halting
```

--- a/biscuit/GNUmakefile
+++ b/biscuit/GNUmakefile
@@ -85,9 +85,9 @@ SKELDEPS := $(shell find $(SKEL))
 
 CPUS := $(shell echo $${CPUS:-1})
 
-QOPTS := -m 8G -smp cpus=$(CPUS) -cpu Nehalem
+QOPTS := -m 8G -smp cpus=$(CPUS) -cpu host
 # for testing bhw2's configuration
-#QOPTS := -m 8G -cpu Nehalem -smp sockets=2,cores=10,threads=2
+#QOPTS := -m 8G -cpu host -smp sockets=2,cores=10,threads=2
 
 OS := $(shell uname -s)
 

